specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json

```diff
@@ -0,0 +1,946 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Local Search Client",
+    "description": "The Local Search client lets you send a search query to Bing and get back search results that include local businesses such as restaurants, hotels, retail stores, or other local businesses. The query can specify the name of the local business or it can ask for a list (for example, restaurants near me).",
+    "version": "1.0"
+  },
+  "parameters": {
+    "x-bingapis-sdk": {
+      "name": "X-BingApis-SDK",
+      "description": "Activate swagger compliance",
+      "x-ms-parameter-location": "method",
+      "required": true,
+      "type": "string",
+      "in": "header",
+      "x-ms-enum": {
+        "name": "XBingApisSDK",
+        "modelAsString": true
+      },
+      "enum": [
+        "true"
+      ]
+    }
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/localbusinesses/v7.0/search",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/localbusinesses/v7.0/search": {
+      "get": {
```

Comment:
Please, provide an example for the operation.

Reply:
just added x-ms-examples. See commit https://github.com/Azure/azure-rest-api-specs/pull/4349/commits/682b78525dccd7bfd44e60bfaa553a15f166e355.